docs/development.md

```diff
@@ -430,6 +430,31 @@ You can optionally set the following variables:
 
 You can also customize the configuration of the CAPZ cluster (assuming that `SKIP_CREATE_WORKLOAD_CLUSTER` is not set). See [Customizing the cluster deployment](#customizing-the-cluster-deployment) for more details.
 
+#### Conformance Testing
```

Comment:
can you make this changes in https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/book/src/developers/development.md instead? I'm removing the duplicate in #1356